charts/kserve-resources/templates/configmap.yaml

```diff
@@ -321,6 +324,15 @@ data:
 
            # cpuLimit is the limits.cpu to set for the batcher container.
            "cpuLimit": "1"
+
+           # maxBatchSize is the default maximum batch size for batcher.
+           "maxBatchSize": "32",
+
+           # maxLatency is the default maximum latency for batcher.
+           "maxLatency": "5000",
+
+           # timeout is the default timeout for batcher.
+           "timeout": "100"
```

Comment:
This is not actually a timeout, it is used internally for the select loop as a time tick. I think this should not be exposed to user.

Reply:
Removed `timeout`